internal/cmd/templatefuncs.go

```diff
@@ -73,6 +73,8 @@ func (c *Config) lookPathTemplateFunc(file string) string {
 		return path
 	case errors.Is(err, exec.ErrNotFound):
 		return ""
```

Comment:
I think the above two lines can be removed.

Reply:
the exec.ErrNotFound? No, it's expected too:

https://github.com/golang/go/blob/master/src/os/exec/lp_unix.go#L58

Reply:
BTW and FYI: https://github.com/golang/go/pull/49492

Reply:
Awesome, thanks!